### PR TITLE
Service for Name variable  + TodoList template task for 28.04 done

### PR DESCRIPTION
**Pull request by EvaMalinina**

### CODE REVIEW

File: todo-app/src/app/todo.ts [hunk 1]
```diff
@@ -0,0 +1,5 @@
+export interface Todo {
+  id?: number;
+  title?: string;
+  isDone?: boolean;
+}
```
Comment (torin4fan) [inline todo-app/src/app/todo.ts]: 1. Rename Todo interface to TodoInterface
2. Create Class Todo with default values based on TodoInterface
3. Create a folder 'Models'. Rename todo.ts on todo.model.ts and move this file to "Models" folder

File: todo-app/src/app/todo-list/todo-list.component.ts [hunk 2]
```diff
@@ -0,0 +1,44 @@
...
+    });
+  }
+
+  listTodo() {
+    this.todoDataService.listTodo().subscribe((data: Todo[])=>{
+      this.todos = data;
+    });
+  }
+
+  mark(todo) {
```
Comment (torin4fan) [inline todo-app/src/app/todo-list/todo-list.component.ts]: Use typing were it possible. For example `mark(todo: Todo): void` etc

File: todo-app/src/app/todo-list/todo-list.component.ts [hunk 3]
```diff
@@ -0,0 +1,44 @@
...
+
+@Component({
+  selector: 'app-todo-list',
+  templateUrl: './todo-list.component.html',
+  styleUrls: ['./todo-list.component.scss']
+})
+export class TodoListComponent implements OnInit {
+
+  todos: Todo[];
+  todo: Todo = {
```
Comment (torin4fan) [inline todo-app/src/app/todo-list/todo-list.component.ts]: Better to use the Class from model

File: todo-app/src/app/todo-data.service.ts [hunk 4]
```diff
@@ -0,0 +1,32 @@
+import { Injectable } from '@angular/core';
```
Comment (torin4fan) [inline todo-app/src/app/todo-data.service.ts]: Please create a separate folder for services

**Review by torin4fan — APPROVED** [review]

File: todo-app/src/app/todo.ts (hunk 1 above)
Comment (EvaMalinina) [reply todo-app/src/app/todo.ts]: Done.

**Comment by EvaMalinina** [comment]
everything done

**Pull request closed without merging**